akka-docs/src/main/paradox/scala/common/cluster.md

```diff
@@ -176,11 +176,10 @@ seen set (have seen the new state) then the cluster gossips 3 times instead of o
 every second. This adjusted gossip interval is a way to speed up the convergence process
 in the early dissemination phase after a state change.
 
-The choice of node to gossip with is random but it is biased to towards nodes that
-might not have seen the current state version. During each round of gossip exchange when
-no convergence it uses a probability of 0.8 (configurable) to gossip to a node not
-part of the seen set, i.e. that probably has an older version of the state. Otherwise
-gossip to any random live node.
+The choice of node to gossip with is random but biased towards nodes that might not have seen
+the current state version. During each round of gossip exchange, when convergence is not yet reached, a node
+uses a configurable probability of 0.8 to gossip with another node which is not part of the seen set, i.e. 
+which is likely to have an older version of the state. Otherwise it gossips with any random live node.
```

Comment:
AFAIK "gossip with someone about something" - see "Examples of gossip in a Sentence
" [here](https://www.merriam-webster.com/dictionary/gossip)